redash/handlers/query_results.py

```diff
@@ -96,6 +96,15 @@ def run_query(query, parameters, data_source, query_id, max_age=0):
         return {'job': job.to_dict()}
 
 
+def get_download_filename(query_result, query=None, filetype='json'):
```

Comment:
Why are you defaulting `filetype` to `json`? That's a handler concern and is already done there.